bundle.yaml

```
@@ -1,20 +1,23 @@
 bundle: kubernetes
 name: kubeflow-pipelines
 applications:
-  argo-controller:        { charm: ch:argo-controller, channel: latest/edge, scale: 1 }
-  minio:                  { charm: ch:minio, channel: latest/edge,       scale: 1 }
-  kfp-api:                { charm: ch:kfp-api, channel: latest/edge,                        scale: 1 }
-  kfp-db:                 { charm: cs:~charmed-osm/mariadb-k8s-35,    scale: 1, options: { database: mlpipeline } }
-  kfp-persistence:        { charm: ch:kfp-persistence, channel: latest/edge,                scale: 1 }
-  kfp-schedwf:            { charm: ch:kfp-schedwf, channel: latest/edge,                    scale: 1 }
-  kfp-ui:                 { charm: ch:kfp-ui, channel: latest/edge,                         scale: 1 }
-  kfp-viewer:             { charm: ch:kfp-viewer, channel: latest/edge,                     scale: 1 }
-  kfp-viz:                { charm: ch:kfp-viz, channel: latest/edge,                        scale: 1 }
+  argo-controller:         { charm: ch:argo-controller, channel: latest/edge, scale: 1 }
+  metacontroller-operator: { charm: ch:metacontroller-operator, channel: latest/edge, scale: 1 }
```

Fixes "charm needs trust" error (metacontroller needs trust to create RBAC)
```suggestion
  metacontroller-operator: { charm: ch:metacontroller-operator, channel: latest/edge, scale: 1, trust: true }
```